mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -167,7 +165,7 @@ public Future<List<Record>> parseRawRecordsChunkForJobExecution(RawRecordsDto ch
       .onSuccess(parsedRecords -> {
         fillParsedRecordsWithAdditionalFields(parsedRecords);
 
-        if (updateMarcActionExists(jobExecution) || updateInstanceActionExists(jobExecution) || isCreateOrUpdateItemOrHoldingsActionExists(jobExecution, parsedRecords)) {
```

Comment:
We cannot delete these conditions, especially for Poppy - it will break many other flows. You could probably just add another OR clause with the check for MARC Authority profile